### PR TITLE
[HOLD FOR .NET 6 RELEASE] - Update desktop book to .NET 6

### PR DESCRIPTION
**Pull request by OliaG**
## Summary

Describe your changes here.

Fixes #Issue_Number (if available)

### CODE REVIEW

**Review by IEvangelist — COMMENTED** [review]
A few nits, otherwise this looks great. Thank you @OliaG, happy to :shipit: once updated.

**Review by IEvangelist — APPROVED** [review]
Looks good to me, we'll :shipit: around the .NET 6 release - thank you 🙏

**Review by gewarren — COMMENTED** [review]
I left a few additional suggestions.

File: docs/architecture/modernize-desktop/index.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
 ---
-title: Modernizing Desktop Apps on Windows 10 with .NET 5
-description: Learn how to modernize existing desktop apps with .NET 5
-ms.date: 01/06/2021
+title: Modernizing Desktop Apps on Windows 10 with .NET 6
+description: Learn how to modernize existing desktop apps with .NET 6
+ms.date: 10/24/2021
 ---
 # Modernizing Desktop Apps on Windows 10 with .NET 5
```
Comment (IEvangelist) [inline docs/architecture/modernize-desktop/index.md:6]:
```suggestion
# Modernizing Desktop Apps on Windows with .NET 6
```

File: docs/architecture/modernize-desktop/index.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
 ---
-title: Modernizing Desktop Apps on Windows 10 with .NET 5
-description: Learn how to modernize existing desktop apps with .NET 5
-ms.date: 01/06/2021
+title: Modernizing Desktop Apps on Windows 10 with .NET 6
```
Comment (IEvangelist) [inline docs/architecture/modernize-desktop/index.md:2]:
```suggestion
title: Modernizing Desktop Apps on Windows with .NET 6
```

File: docs/architecture/modernize-desktop/toc.yml [hunk 3]
```diff
@@ -1,4 +1,4 @@
-- name: Modernizing desktop apps on Windows 10 with .NET 5
+- name: Modernizing desktop apps on Windows 10 with .NET 6
```
Comment (IEvangelist) [inline docs/architecture/modernize-desktop/toc.yml:1]:
```suggestion
- name: Modernizing Desktop Apps on Windows with .NET 6
```

**Review by andy7439 — COMMENTED** [review]
Deploying Modern Desktop

**Comment by andy7439** [comment]
https://github.com/dotnet/docs/pull/26643#pullrequestreview-1105551000

**Comment by andy7439** [comment]
https://github.com/dotnet/docs/pull/26643#pullrequestreview-1105551000

**Pull request merged**